src/editor/EditorCommandHandlers.js

```diff
@@ -32,7 +32,7 @@ define(function (require, exports, module) {
         Strings            = require("strings"),
         Editor              = require("editor/Editor").Editor,
         CommandManager     = require("command/CommandManager"),
-		Dialogs        	   = require("widgets/Dialogs"),
+	Dialogs        	   = require("widgets/Dialogs"),
```

Comment:
this still uses a tab. Please set your code editor to use spaces for tabs in this file. If you do a text selection on any of the lines above/below you'll see that it's indentation that purely uses spaces, unlike your line here, which uses a tab